resources/views/components/button/enhanced.blade.php

```diff
@@ -0,0 +1,12 @@
+{{--
+This is the enhanced button the button can be used for the add to cart buttons
+and it can be used inside the checkout.
+Example:
+
```

Comment:
Maybe rewrite this a little. It's a bit vague.

The enhanced button is a variant specifically used for anything conversion related. Like purchasing a product or completing the checkout.